src/Workspaces/CSharpTest/Formatting/FormattingTests.cs

```diff
@@ -6705,7 +6704,20 @@ class Program
     }
 }
 ";
-            await AssertFormatAsync(code, code);
+            var expected = @"
```

Comment:
tag @sharwell to check if this test case is OK. I am not fully understand the original scenario.

Reply:
This change would be a regression for the scenario in #38224. Lines declaring a parameter are allowed to have a leading block comment without altering the position of the first non-whitespace character.

Reply:
It would be good to add a test where the parameters in this test case are not initially indented, but become indented.

Reply:
Aright, I will update the test later